crates/zkvm/src/lib.rs

```
@@ -127,7 +127,7 @@ mod tests {
                     }
                     let journal = Journal::abi_decode(&info.journal.bytes)?;
                     assert_eq!(
-                        from_wormhole_address(journal.emitterContract),
+                        from_wormhole_address(journal.emitterContract).unwrap_or(Address::ZERO),
```

Better to `unwrap` or `expect` here. This is a silent failure otherwise

---

Roger, will revise!